cmd/daemon.go

```diff
@@ -27,17 +28,17 @@ func daemonCommand(c *cli.Context) error {
 		return err
 	}
 
-	if envcfg.Daemon.Listen == "" {
-		logging.S().Fatal("missing daemon configuration. copy env-example.yaml to .env.toml and configure [daemon] section")
-	}
-
-	srv := server.New(envcfg.Daemon.Listen)
+	var (
+		listen = envcfg.Daemon.Listen
+		srv    = server.New(listen)
+	)
 
 	done := make(chan os.Signal, 1)
 	signal.Notify(done, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
 
 	go func() {
-		logging.S().Infow("listen and serve", "addr", srv.Addr)
+		fmt.Printf("daemon listening on addr: %s\n", srv.Addr)
```

Comment:
Why `fmt.Printf` and not logging? We are lacking the timestamp for example, which would be useful when we run this as a service.

Also we are substituting structured logging for non-structured, which brings less value.

Reply:
Because I want this to be printed out unconditionally, and we run the logger with `WARN` level by default.

Reply:
We both agree that logging can be a religious topic, just sharing my opinion on this religion, and maybe we can reach a common ground where both are happy and logging is sane for a large amount of people :)